phalcon/mvc/model/query.zep

```diff
@@ -178,6 +180,9 @@ class Query implements QueryInterface, InjectionAwareInterface
 		} else {
 			let this->_enableImplicitJoins = globals_get("orm.enable_implicit_joins");
 		}
+
+		let this->_nestingLevel = -1;
```

Comment:
How about setting `-1` as the default value when you define the property on the class?

Reply:
Absolutely!